extensions/control-plane/api/management-api/policy-definition-api/src/test/java/org/eclipse/edc/connector/api/management/policy/validation/PolicyDefinitionValidatorTest.java

```diff
@@ -68,4 +71,16 @@ void shouldFail_whenPolicyIsMissing() {
                 .filteredOn(it -> EDC_POLICY_DEFINITION_POLICY.equals(it.path()))
                 .anySatisfy(violation -> Assertions.assertThat(violation.message()).contains("mandatory"));
     }
+
+    @Test
+    void shouldFail_whenTypeIsMissing() {
```

Comment:
it should also fail when type is not `Set`, maybe a dummy one can be added in this test